src/parse_file.cpp

```diff
@@ -7,16 +7,31 @@
 // identity
 #include "orc/parse_file.hpp"
 
+// application config
+#include "orc/features.hpp"
+
 // stdc++
 #include <bit>
 #include <cstdio>
 
 // system
 #include <fcntl.h> // open
+#include <sys/mman.h>
+#include <unistd.h> // close#if ORC_FEATURE(MACH_O)
```

Comment:
This comment doesn't make sense? It was never open, was it? (or is unrelated to mach-o?)

Reply:
Sorry - this is a copy/paste error on my part. I'm using Ubuntu on a VM for this PR, and I keep getting caught up by it's hotkeys/shortcuts.